packages/composerize/__tests__/mappings.test.js

```diff
@@ -168,3 +169,771 @@ test('--ulimit (https://github.com/magicmark/composerize/pull/262)', () => {
+                    device_write_bps:
+                        -
+                            path: /dev/sda
+                            rate: 1mb
+                    device_write_iops:
+                        -
+                            path: /dev/sda
+                            rate: 1000
+                image: ubuntu"
+            `);
+});
+
+test('blkio 2 device', () => {
+    expect(Composerize('docker run -it --blkio-weight 16 --blkio-weight-device "/dev/sda:200"  --blkio-weight-device "/dev/sdb:300" --device-read-bps=/dev/sda:1mb --device-read-bps=/dev/sdb:2mb --device-read-iops=/dev/sda:1000 --device-write-bps=/dev/sda:1mb --device-write-iops=/dev/sda:1000 ubuntu'))
+        .toMatchInlineSnapshot(`
+        "version: '3.3'
+        services:
+            ubuntu:
+                stdin_open: true
+                tty: true
+                blkio_config:
+                    weight: 16
+                    weight_device:
+                        -
+                            path: /dev/sda
+                            weight: 200
+                        -
+                            path: /dev/sdb
+                            weight: 300
+                    device_read_bps:
+                        -
+                            path: /dev/sda
+                            rate: 1mb
+                        -
+                            path: /dev/sdb
+                            rate: 2mb
+                    device_read_iops:
+                        -
+                            path: /dev/sda
+                            rate: 1000
+                    device_write_bps:
+                        -
+                            path: /dev/sda
+                            rate: 1mb
+                    device_write_iops:
+                        -
+                            path: /dev/sda
+                            rate: 1000
+                image: ubuntu"
+            `);
+});
+
+test('--device-cgroup-rule', () => {
+    expect(Composerize('docker run -d --device-cgroup-rule="c 42:* rmw" --name my-container my-image'))
+        .toMatchInlineSnapshot(`
+        "version: '3.3'
+        services:
+            my-image:
+                device_cgroup_rules:
+                    - 'c 42:* rmw'
+                container_name: my-container
+                image: my-image"
+            `);
+});
+
+test('--healthcheck-cmd ', () => {
+    const command =
+        'docker run --health-cmd=healthcheck.sh --health-interval=60s --health-timeout=10s --health-start-period=30s --health-retries=2  nginx:latest';
+
+    expect(Composerize(command)).toMatchInlineSnapshot(`
+        "version: '3.3'
+        services:
+            nginx:
+                healthcheck:
+                    test: healthcheck.sh
+                    interval: 60s
+                    timeout: 10s
+                    start_period: 30s
+                    retries: '2'
+                image: 'nginx:latest'"
+      `);
+});
```

Comment:
hmm github doesn't like the line ending here

try running eslint? https://github.com/magicmark/composerize/blob/ff59318b1d6318d1d88616f6f1f77a5f3904035d/package.json#L13

Reply:
Did it, will retry, may be not the same version

Reply:
eslint made